### ISSUE TITLE
embed field lacks info

### ISSUE THREAD

**Issue by Boorj** [bolt/docs#928]
When i'm trying to use `embed` and paste code from youtube (`<iframe...>..`) - it gives and error:
![image](https://user-images.githubusercontent.com/2145313/44583759-e6156c80-a7d0-11e8-86b4-416fcbf054e1.png)

My first thought was that `embed` field should allow using some lines of html-code.

here's value:
![image](https://user-images.githubusercontent.com/2145313/44583771-f1689800-a7d0-11e8-92c9-f06820e5fcf2.png)

Maybe we should add some examples? Or write down requirements (i know about placeholder, yes)

**Comment by eduardomart** [comment]
But it already says '***URL*** of content to embed'
If you wanna take the embed code that sites like YouTube give you, you could simple use a `type: textarea` instead

**Comment by Boorj** [comment]
I agree, but.. on many services when someone is you have "embed code" . So i thought it deserves a mention in docs: https://docs.bolt.cm/3.5/fields/embed
like "you can only paste url, not code"

**Comment by Boorj** [comment]
Or even better:
> Field is supposed to get an _url_ as input. If the url is valid, Bolt gets data from the service (i.e. from youtube for `http://youtube.com/...` and provides that to record. List of received array fields depends on service. See more information on [official oEmbed website](https://oembed.com/)
